.gitignore

```diff
@@ -12,6 +12,12 @@
 /python/_venv*
 /python/tidy/servo_tidy.egg-info
 /tests/wpt/sync
+/tests/wpt/meta/.cache
+/tests/wpt/meta-legacy-layout/.cache
+/tests/wpt/mozilla/meta/.cache
+/tests/wpt/mozilla/meta-legacy-layout/.cache
+/tests/wpt/webgl/meta/.cache
+/tests/wpt/webgl/meta-legacy-layout/.cache
```

Comment:
NIT: Maybe we can just ignore .cache folders.

Reply:
I was worried about it being too general and hiding stray files that we should be aware of. In [573213a](https://github.com/servo/servo/commit/573213a7559b31073e83c90f45850851084ff1bf) I've change this to use wildcards, but still scoped to `/tests/wpt/`.